packages/flutter_tools/templates/app/ios-objc.tmpl/Runner.xcodeproj/xcshareddata/xcschemes/Runner.xcscheme.tmpl

```diff
@@ -1,121 +0,0 @@
-<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Odd we had an Objective-C and Swift version of schemes, they were identical except for a single `BlueprintIdentifier`.